api/app/change_log/tracker.py

```diff
@@ -29,9 +30,22 @@ def store_state(self):
         Stores the current state of an instance. Is called in the ChangeLogger when initializing the logger.
         """
         for field in self.instance._meta.get_fields():
-            if isinstance(field, (models.ManyToManyRel, models.ManyToOneRel)):
+            if field.name in ['created_at', 'updated_at']:
```

Comment:
This is SIA specific naming, should not be in the `change_log` reusable app.

Reply:
It is no longer there